installation/cmd/run.ps1

```diff
@@ -1,15 +1,25 @@
 param (
     [string]$CR_PATH = "",
     [switch]$SKIP_MINIKUBE_START = $false,
+    [switch]$KNATIVE = $false,
     [string]$VM_DRIVER = "hyperv"
 )
 
 $CURRENT_DIR = Split-Path $MyInvocation.MyCommand.Path
 $SCRIPTS_DIR = "${CURRENT_DIR}\..\scripts"
 $DOMAIN = "kyma.local"
+$MINIKUBE_EXTRA_ARGS = ""
+$CREATE_CR_EXTRA_ARGS = ""
+
+if ($KNATIVE -eq $false) {

```

Comment:
This seems to work differently than in bash script. In `run.sh` you apply the knative cr when KNATIVE variable is non-zero, here you run this part when KNATIVE is false, which is default value for this flag.